fix tests in lite/kernels/internal

freedomtan opened the pull request:
make
```
bazel test --config opt //tensorflow/contrib/lite/kernels/internal:all
```
work. So that
```
bazel test --config opt //tensorflow/contrib/lite/kernels/...
```
works.


This is a sequel to https://github.com/tensorflow/tensorflow/pull/17700

freedomtan committed:
fix tests in lite/kernels/internal

make
```
bazel test -config opt //tensorflow/contrib/lite/kernels/internal:all
```
work. So that
```
bazel test -config opt //tensorflow/contrib/lite/kernels/...
```
works

andrehentz commented on tensorflow/contrib/lite/kernels/internal/log_quantized_test.cc:
```diff
@@ -69,7 +69,7 @@ inline int32 LogPositiveValuesViaFloat(int32 input_val, int input_integer_bits,
                      const std::vector<int32>& reference_output,
                      const std::vector<int32>& test_input,
-                     const string& check_label, int input_integer_bits,
+                     const std::string& check_label, int input_integer_bits,
```
For the string issues, it is probably better to include contrib/lite/string.h

freedomtan replied:
OK, will do.

freedomtan committed:
use tensorflow/contrib/lite/string.h

as suggested, use //tensorflow/contrib/lite:string instead of
using std::string explicitly

tensorflowbutler commented:
Nagging Assignee @yifeif: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
